toys/release/.data/rad-yard-templates/default/layout/yaml/format.rb

```diff
@@ -161,6 +161,7 @@ def code_tail
 end
 
 def codeblock str
+  str = str.gsub("&", "&amp;").gsub("<", "&lt;").gsub(">", "&gt;")
```

Comment:
Should we create a generic method that can sanitise all such html characters? We can call that from anywhere that way, when needed.

Reply:
I'm inclined not to think too much about it right now, as we're likely to overhaul it again anyway. This entire yard template needs a general refactoring pass. (It grew organically and fairly explosively, and there are a lot of concerns that should be encapsulated in separate classes. And I'm sure there's still a fair amount of dead code from previous abandoned experiments.)